turbinia/lib/libcloudforensics.py

```diff
@@ -535,15 +623,12 @@ def get_value(self, key):
     """Get specific value from the resource key value store.
 
     Args:
-      key: Key to get value from.
+      key: A key of type String to get key's corresponding value.
 
     Returns:
       Value of key or None if key is missing.
     """
-    if not self._data:
-      operation = self.get_operation().execute()
-      self._data = self.project.gce_operation(
-          operation, zone=self.zone, block=False)
+    self._data = self.get_operation().execute()
     return self._data.get(key)
 
   def get_source_string(self):
```

Comment:
Wouldn't `get_resource_url` be a better name for this method?

Reply:
It it definitely a better name, however, get_source_string and get_value aren't methods that I added, these two methods specifically were already there, I just did changes. get_source_string is referenced in create_disk_from_snapshot and attach_disk methods. It might be referenced in other modules importing the libcloudforensics library as well, I am afraid I would break things if I change this. Should we leave it, add a new one and deprecate it or what do you think?

Reply:
Let's leave it like this for now. Thanks!